templates/home/captures.html

```diff
@@ -0,0 +1,168 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati Captures &#10084; {% endblock %}
+
+{% block style %}
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css" integrity="sha512-P5MgMn1jBN01asBgU0z60Qk4QxiXo86+wlFahKrsQf37c9cro517WzVSPPV1tDKzhku2iJ2FVgL67wG03SGnNA==" crossorigin="anonymous" />
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js" integrity="sha512-894YE6QWD5I59HgZOGReFYm4dnWc1Qt5NtvYSaNcOP+u1T9qYdvdihz0PPSiiqn/+/3e7Jo4EaG7TubfWGUrMQ==" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.3/css/lightbox.min.css" integrity="sha512-ZKX+BvQihRJPA8CROKBhDNvoc2aDMOdAlcm7TUQY+35XYtrd3yh95QOOhsPDQY9QnKE0Wqag9y38OIgEvb88cA==" crossorigin="anonymous" />
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.3/js/lightbox.min.js" integrity="sha512-k2GFCTbp9rQU412BStrcD/rlwv1PYec9SNrkbQlo6RZCf75l6KcC3UwDY8H5n5hl4v77IDtIPwOk9Dqjs/mMBQ==" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+<style type="text/css">
+
+    .home-btn {
+        background-color: #f7f3e9;
+        color: #000000;
+        border: none;
+        border-radius: 20px;
+    }
+    body{
+        background-color:#f7f3e9;
+    }
+    .img-fluid{
+        margin-bottom: 15px;
+    }
+
+    .event-name{
+        padding:40px 0px;
+        text-align: left;
+    }
+    
+    .event-sections{
+        margin-top:10px;
+    }
+    .captures-page-heading{
+        font-size:35px;
+        font-weight:bolder;
+    }
```

Comment:
You can remove `font-size` from here as it doesn't seem to do anything anyways (`h1`'s default font size overrides this, which is infact well-sized. And instead of `font-weight`, you can use `font-weight-bold` class from bootstrap.